src/main/java/de/guntram/mcmod/durabilityviewer/client/gui/GuiItemDurability.java

```diff
@@ -183,6 +184,9 @@ public void onRenderGameOverlayPost(DrawContext context, float partialTicks) {
         ItemIndicator[] trinkets;
         if (haveTrinketsApi) {
             List<ItemStack> equipped = getTrinkets(player);
```

Comment:
I'm not sure where exactly `getTrinkets()` gets implemented, but wouldn't it be better to return the empty list there instead of null?

````diff
    public List<ItemStack> getTrinkets(LivingEntity player) {
        Optional<TrinketComponent> component = TrinketsApi.getTrinketComponent(player);
-        return null; // component.map(trinketComponent -> trinketComponent.getEquipped(prdct -> true).stream().map(Pair::getRight).toList()).orElse(null);
+        return List.of(); // component.....orElse(List.of());
    }
````